test/Microsoft.AspNetCore.Mvc.FunctionalTests/ApiExplorerTest.cs

```diff
@@ -394,6 +394,26 @@ public ApiExplorerTest(MvcTestFixture<ApiExplorerWebSite.Startup> fixture)
         }
 
         [Theory]
+        [InlineData("GetVoidNoContent")]
+        [InlineData("GetTaskNoContent")]
+        public async Task ApiExplorer_ResponseType_VoidWithoutAttribute204NoContent(string action)
+        {
+            // Arrange & Act
+            var response = await Client.GetAsync(
+                "http://localhost/ApiExplorerResponseTypeWithoutAttribute/" + action);
```

Comment:
It was my bad to put in the actions having the `ProducesResponseType` into this controller which says 'without'. Could you move these new actions into the controller which says 'ApiExplorerResponseTypeWithAttributeController.cs' as they have explicit attributes over them.
Can we rename the actions too? example: GetVoidNoContent => GetVoidWithExplicitResponseTypeStatusCode